.github/workflows/test.yml

```diff
@@ -50,13 +50,13 @@ jobs:
         set-safe-directory: true
 
     - name: Setup Node.js environment
-      uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+      uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
```

Comment:
Maybe dependabot is not updating them because of the hash? Or because of the comment. Not sure either :man_shrugging:

Reply:
Dependabot is smart enough for it to work on my other projects. If the comment is wrong, it'll also override it (rather than ignoring it or not updating), which is nice.